README.md

```diff
@@ -36,70 +71,49 @@ Public Invidious instances are listed on the documentation website: https://inst
 - No Contributor license Agreement
 - Available in many languages, thanks to [Weblate](https://hosted.weblate.org/projects/invidious/)
 
----
 
-## Screenshots:
+## Documentation
 
-| Player                                                                                                                  | Preferences                                                                                                             | Subscriptions                                                                                                               |
-| ----------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------- |
-| [<img src="screenshots/01_player.png?raw=true" height="140" width="280">](screenshots/01_player.png?raw=true)           | [<img src="screenshots/02_preferences.png?raw=true" height="140" width="280">](screenshots/02_preferences.png?raw=true) | [<img src="screenshots/03_subscriptions.png?raw=true" height="140" width="280">](screenshots/03_subscriptions.png?raw=true) |
-| [<img src="screenshots/04_description.png?raw=true" height="140" width="280">](screenshots/04_description.png?raw=true) | [<img src="screenshots/05_preferences.png?raw=true" height="140" width="280">](screenshots/05_preferences.png?raw=true) | [<img src="screenshots/06_subscriptions.png?raw=true" height="140" width="280">](screenshots/06_subscriptions.png?raw=true) |
-
----
+The complete documentation is available on https://docs.invidious.io/ (or alternatively on its own [Github repository](https://github.com/iv-org/documentation)).
 
-## Donate:
 
-Bitcoin (BTC): [bc1qfhe7rq3lqzuayzjxzyt9waz9ytrs09kla3tsgr](bitcoin:bc1qfhe7rq3lqzuayzjxzyt9waz9ytrs09kla3tsgr)
+## Extensions
 
-Monero (XMR): [41nMCtek197boJtiUvGnTFYMatrLEpnpkQDmUECqx5Es2uX3sTKKWVhSL76suXsG3LXqkEJBrCZBgPTwJrDp1FrZJfycGPR](monero:41nMCtek197boJtiUvGnTFYMatrLEpnpkQDmUECqx5Es2uX3sTKKWVhSL76suXsG3LXqkEJBrCZBgPTwJrDp1FrZJfycGPR)
+[Extensions](https://docs.invidious.io/Extensions.md) can be found in the wiki, as well as documentation for integrating it into other projects.
 
----
 
-## Documentation:
+## Contribute
 
-The complete documentation is available on https://docs.invidious.io/ (or alternatively on its own [Github repository](https://github.com/iv-org/documentation)).
+### Code
 
----
+1.  Fork it ( https://github.com/iv-org/invidious/fork ).
+2.  Create your feature branch (git checkout -b my-new-feature).
+3.  Commit your changes (git commit -am 'Add some feature').
```

Comment:
`git commit -am` doesn't work if new files are created, we should instead say:

Add all your changes (`git add .`): 

Commit your changes (`git commit -m "Add some feature").

Also let's put all the git commands in "code block" (`stuff`).

Reply:
sure!